readme.md

```diff
@@ -158,6 +158,7 @@ Made with Electron.
 - [Dext](https://github.com/vutran/dext) - Launcher.
 - [PB for Desktop](https://github.com/sidneys/pb-for-desktop) - Pushbullet client.
 - [GroupMe](https://github.com/dcrousso/GroupMe) - Unofficial GroupMe app.
+- [ndm](https://github.com/720kb/ndm) - Npm desktop manager
```

Comment:
@sindresorhus yeah sorry, I thought I had read that inside the guidelines, like the description should start with a capital letter, my bad (makes enough sense also to me to read Npm )

Gonna rebase and push the changes, sorry for wasting time on it.